ores/score_processors/celery.py

```diff
@@ -102,7 +112,21 @@ def _generate_id(self, context, model, rev_id):
 
         return ":".join(str(v) for v in [context, model, rev_id, version])
 
+    def _queue_full(self):
+        # Check redis to see if the queue is too big.  If not, lump it on!
+        # This will result in a race condition, but it should have OK
+        # properties
+        if self.redis is not None:
+            queue_size = self.redis.llen("celery")
```

Comment:
I've looked everywhere I could think of and I couldn't figure out how to get the name of the default queue.